Vector.hpp

```diff
@@ -222,25 +204,29 @@ class Vector
 
 
 private:
-	T* _dynamicVector = nullptr;
-	size_t _capacity = 0;
-	size_t _size = 0;
+	T* _dynamic_vector;
+	size_t _capacity;
+	size_t _size;
+	static constexpr int INITALIZE_SIZE = 2;
 
-	void Realloc(size_t newCapacity)
-	{
 	
+	///@brief calculate the new capacity  
+	/// @returns The new capacity, which is the current capacity increased by 50%
+	size_t calculateResizeSize() const {
+		return _capacity + _capacity / 2;
```

Comment:
nice use of helper functions
if I were you, I'd still put the `2` inside a `constexpr`